internal/config/config.go

```diff
@@ -10,6 +11,11 @@ import (
 	"github.com/gopasspw/gopass/pkg/gitconfig"
 )
 
+const (
+	DefaultLength     = 24
+	DefaultXKCDLength = 4
```

Comment:
Do you really need to move that one here?

Reply:
Yes because in need it inside `DefaultLengthFromEnv` function and I may have import problem if I leave it in the former package. In addition, it sound more like a config part.

Reply:
Since it's exported, how about using this one in the former package and removing it there then?